### PR TITLE
adding cabin

### PR DESCRIPTION
**Pull request by puja108**

### CODE REVIEW

**Comment by puja108** [comment]
RFR @marians

File: content/guides/using-the-cabin-ios-app/index.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
++++ title = "Using the Cabin iOS App" description = "You can use the Cabin iOS ap to access and control your Kubernetes cluster." date = "2016-10-28" type = "page" weight = 100 categories = ["guides"] +++
```
Comment (marians) [inline content/guides/using-the-cabin-ios-app/index.md]: ap -> app

File: content/guides/using-the-cabin-ios-app/index.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+# Using the Cabin iOS App
+
+You can use the free [Cabin iOS app](https://appsto.re/fr/4kHXdb.i) from [skippbox](http://www.skippbox.com/cabin/) to acces your cluster from your iOS device.
```
Comment (marians) [inline content/guides/using-the-cabin-ios-app/index.md]: Let's use the full URL. Probably living longer.

https://itunes.apple.com/fr/app/cabin-manage-kubernetes-applications/id1137054392

**Comment by puja108** [comment]
done RFR

**Pull request merged**